lib/app-components/addon/components/project-contributors/list/template.hbs

```diff
@@ -1,28 +1,33 @@
-<div class='row hidden-xs'>
-    <div class='col-sm-3 col-sm-offset-2'>
-        <strong>{{t 'app_components.project_contributors.list.name'}}</strong>
-    </div>
-    <div class='col-sm-3'>
-        <strong>{{t 'app_components.project_contributors.list.permissions'}}</strong>
-        <span>
+{{#if this.isNotMobile}}
+    <div local-class='header-container'>
+        <div local-class='icon-spacer-container'></div>
+        <div local-class='name-container'>
+            <strong>{{t 'app_components.project_contributors.list.name'}}</strong>
+        </div>
+        <div local-class='permission-container'>
+            <strong>{{t 'app_components.project_contributors.list.permissions'}}</strong>
             <FaIcon @icon='question-circle' />
             <EmberPopover @side='bottom' @popperContainer='body'>
-                <h3>{{t 'app_components.project_contributors.list.permissions_popover_title'}}</h3>
-                {{t 'app_components.project_contributors.list.permissions_popover' htmlSafe=true}}
+                <div class='text-left'>
```

Comment:
So `<div local-class='text-left'>` didn't work here? Also, do we want some Ember Popovers to be left justified and others not?